pkg/crc/logging/stdin_hook.go

```diff
@@ -0,0 +1,48 @@
+package logging
+
+import (
+	"io"
+	"os"
+
+	"github.com/sirupsen/logrus"
+)
+
+// This is stdOutHook to send everything except error and Fatal to standard output.
+type stdOutHook struct {
+	stderr    io.Writer
```

Comment:
Typo here `stderr => stdout`  and this file should be called `stdout_hook.go`, also you could probably make this `Hook stuct` generic and reuse it in all the hooks:
```golang
type logrusHook struct {
      writer       io.Write
      formatter    logrus.Formatter
      level        logrus.Level
}

Reply:
@anjannath we can't reuse the struct since all those struct need to satisfy the `Hook` interface so the method should be part of struct.

```
type Hook interface {
	Levels() []Level
	Fire(*Entry) error
}
```